tests/system/source/main.d

```diff
@@ -25,42 +25,32 @@ import agora.utils.PrettyPrinter;
 import vibe.web.rest;
 
 import std.algorithm;
+import std.exception;
 import std.stdio;
 import core.thread;
 import core.time;
 
-/// Helper struct
-private struct Address
-{
-    ///
-    string host;
-    ///
-    ushort port;
+/// Test node count
+private immutable uint NodeCnt = 3;
 
-    /// Helper function to call make the address vibe.inet.URL friendly
-    public string withSchema () const @safe
+/// Address array of nodes
+private string[] addresses;
+
+void main (string[] args)
+{
+    if (args.length-1 != NodeCnt)
```

Comment:
Why do we force the count to be three ? All this test do is check that a network is consistent.
So we should check that the number of args is >= 2 but I don't think we should limit it to three.

Reply:
That's a very good idea.